buildkite/bazel-bench/bazel_bench.py

```diff
@@ -30,30 +31,26 @@
 import yaml
 
 
-def _platform_path_str(posix_path):
-    """Converts the path to the appropriate format for platform."""
-    if os.name == "nt":
-        return posix_path.replace("/", "\\")
-    return posix_path
-
-
 # TMP has different values, depending on the platform.
 TMP = tempfile.gettempdir()
 PROJECTS = [
     {
         "name": "Bazel",
         "storage_subdir": "bazel",
         "git_repository": "https://github.com/bazelbuild/bazel.git",
-        "bazel_command": "build ...",
-    }
+        "bazel_command": "build //src:bazel",
+        "active": True
+    },
 ]
 BAZEL_REPOSITORY = "https://github.com/bazelbuild/bazel.git"
-DATA_DIRECTORY = _platform_path_str("%s/.bazel-bench/out/" % TMP)
+DATA_DIRECTORY = os.path.join(TMP, ".bazel-bench", "out")
+BAZEL_BENCH_RESULT_FILENAME = "perf_data.csv"
+JSON_PROFILES_AGGR_FILENAME = "json_profiles_aggr.csv"
 
 
 def _bazel_bench_env_setup_command(platform, bazel_commits):
     bazel_bench_env_setup_py_url = (
-        "https://raw.githubusercontent.com/bazelbuild/continuous-integration/master/buildkite/bazel_bench_env_setup.py?%s"
+        "https://raw.githubusercontent.com/bazelbuild/continuous-integration/master/buildkite/bazel-bench/bazel_bench_env_setup.py?%s"
```

Comment:
rather than plain passing the time here, consider using a key=value pair instead so it's future proof when you want to add more args later?

Reply:
@philwo do you happen to know the param for timestamp? Is it `t=...`?

Reply:
I don't really get the conversation here: This is string, what does a key=value pair have to do with it, and what is the "param for timestamp"?

Reply:
I guess what Tobi meant was: instead of `?<timestamp>`, is it possible to do something like `?t=<timestamp>`. My question was whether `t=...` (or something equivalent) is available as a URL query string at all.